### PR TITLE
Rework for Sophos UTM firewall rules and decoders

### PR DESCRIPTION
**Pull request by jcruzlp**
|Related issue|
|---|
|https://github.com/wazuh/wazuh/issues/10972|

## Description

This PR adds the rework for the rules and decoders of Sophos UTM Firewall added in: https://github.com/wazuh/wazuh/pull/10496

This will solve the problem with decoders, as the previous one was developed using the whole processed log instead of the original log, that is present in the `full_log` field as:
```log
<139>2021:05:19-09:39:29 fw-int-name httpd[25917]: [proxy_http:error] [pid 25917:tid 3878202224] (70007)The timeout specified has expired: [client x.x.x.x:60914] AH01102: error reading status line from remote server x.x.x.x:443
```

These kinds of logs are not enough to develop rules and decoders for Sophos, so in this PR the changes will be reverted in this PR, so this PR will only add the form changes to headers for pre-existing rules, decoders and ini testing. And will remove the changes added in the previous PR.

### Syntax

- [x] 1.a Rule tags order must be compliant.
- [x] 1.b Decoder tags order must be compliant.
- [x] 1.c XML blocks must as compact as possible.
- [X] 1.d Only one empty line between rule/decoder and the next rule/decoder.
- [X] 1.e Decoder extracted fields names must use '_' whether a space is needed.
- [X] 1.f Decoder name must use '-' whether a space is needed.
- [X] 2.d XML and ini files must use correct indentation
 
### Grammar

- [X] 2.a Grammar quality.
- [X] 2.b Similar phrases must keep tenses and expressions.
- [X] 2.c Grammar basic rules like capitalization, punctuation marks, sentence construction, and others.

### Semantic

- [x] 3.a New decoder, rule, or test are written in the correct file and grouped correctly inside the file.
- [x] 3.b Group similar rules under same ID's group.
- [x] 3.c: 
     - [x] 3.c1 Find and reuse group name before creating a new one.
     - [x] 3.c2 Include a new group definition in a PR comment.
     - [x] 3.c3 Any group name must use '_' whether it does need a space char.
     - [x] 3.c4 The groups inside a tag must be sorted in alphabetical order.
- [x] 3.d Rule level should be compliant with the [documentation](https://documentation.wazuh.com/current/user-manual/ruleset/rules-classification.html).

### Unit testing

- [X] 4.a A metadata block at the beginning of the .ini file describing software, version, and logs source.
- [X] 4.b Each new rule must have at least one test entry in the correct .ini file.
- [x] 4.c Runtest.py must pass and must include the results in raw format here.
<details><summary>Tests results</summary>
<p>

## python runtests.py -t tests/sophos.ini 

```
- [ File = ./tests/sophos.ini ] ---------
..............

- [ File = ./tests/doas.ini ] ---------
....

- [ File = ./tests/office365.ini ] ---------
................................................................................................................................

- [ File = ./tests/iptables.ini ] ---------
........

- [ File = ./tests/exim.ini ] ---------
.......

- [ File = ./tests/api.ini ] ---------
............................

- [ File = ./tests/trendmicro-cloud-one.ini ] ---------
.................

- [ File = ./tests/web_appsec.ini ] ---------
...............................

- [ File = ./tests/cisco_asa.ini ] ---------
........................................................................................

- [ File = ./tests/firewalld.ini ] ---------
..

- [ File = ./tests/mailscanner.ini ] ---------
.

- [ File = ./tests/nextcloud.ini ] ---------
.......

- [ File = ./tests/icinga.ini ] ---------
....

- [ File = ./tests/ossec.ini ] ---------
.....

- [ File = ./tests/paloalto.ini ] ---------
................

- [ File = ./tests/named.ini ] ---------
.....

- [ File = ./tests/vsftpd.ini ] ---------
....

- [ File = ./tests/gcp.ini ] ---------
...........

- [ File = ./tests/web_rules.ini ] ---------
..........

- [ File = ./tests/sysmon.ini ] ---------
...

- [ File = ./tests/fireeye.ini ] ---------
...

- [ File = ./tests/arbor.ini ] ---------
..

- [ File = ./tests/cisco_ios.ini ] ---------
.................

- [ File = ./tests/sudo.ini ] ---------
........

- [ File = ./tests/modsecurity.ini ] ---------
......

- [ File = ./tests/postfix.ini ] ---------
..

- [ File = ./tests/freepbx.ini ] ---------
......

- [ File = ./tests/SonicWall.ini ] ---------
........

- [ File = ./tests/audit_scp.ini ] ---------
.

- [ File = ./tests/cimserver.ini ] ---------
..

- [ File = ./tests/nginx.ini ] ---------
............

- [ File = ./tests/junos.ini ] ---------
...

- [ File = ./tests/glpi.ini ] ---------
...

- [ File = ./tests/proftpd.ini ] ---------
.......

- [ File = ./tests/syslog.ini ] ---------
......

- [ File = ./tests/fortigate.ini ] ---------
........................................

- [ File = ./tests/f5_big_ip.ini ] ---------
...........................................

- [ File = ./tests/opensmtpd.ini ] ---------
.......

- [ File = ./tests/sophos.ini ] ---------
..............

- [ File = ./tests/netscreen.ini ] ---------
....

- [ File = ./tests/apparmor.ini ] ---------
.....

- [ File = ./tests/squid_rules.ini ] ---------
..

- [ File = ./tests/gitlab.ini ] ---------
.........................

- [ File = ./tests/checkpoint_smart1.ini ] ---------
..................

- [ File = ./tests/apache.ini ] ---------
..................

- [ File = ./tests/dovecot.ini ] ---------
...............

- [ File = ./tests/systemd.ini ] ---------
..

- [ File = ./tests/pam.ini ] ---------
.....

- [ File = ./tests/github.ini ] ---------
....................................................................................................................................................................................................................................................................................................................................

- [ File = ./tests/oscap.ini ] ---------
................................

- [ File = ./tests/huawei_usg.ini ] ---------
...

- [ File = ./tests/cpanel.ini ] ---------
.......

- [ File = ./tests/samba.ini ] ---------
....

- [ File = ./tests/audit_lateral.ini ] ---------
......

- [ File = ./tests/vuln_detector.ini ] ---------
..

- [ File = ./tests/owlh.ini ] ---------
....

- [ File = ./tests/exchange.ini ] ---------
..

- [ File = ./tests/cisco_ftd.ini ] ---------
..........................................

- [ File = ./tests/openvpn_ldap.ini ] ---------
..

- [ File = ./tests/panda_paps.ini ] ---------
........

- [ File = ./tests/fortiauth.ini ] ---------
....

- [ File = ./tests/dropbear.ini ] ---------
...

- [ File = ./tests/mcafee_epo.ini ] ---------
.

- [ File = ./tests/openldap.ini ] ---------
.........

- [ File = ./tests/sshd.ini ] ---------
...........................................

- [ File = ./tests/kernel_usb.ini ] ---------
......

- [ File = ./tests/pix.ini ] ---------
......................

- [ File = ./tests/unbound.ini ] ---------


- [ File = ./tests/rsh.ini ] ---------
..

- [ File = ./tests/cylance.ini ] ---------
.......

- [ File = ./tests/cloudlfare-waf.ini ] ---------
.............

- [ File = ./tests/eset.ini ] ---------
........

- [ File = ./tests/auditd.ini ] ---------
...

- [ File = ./tests/pfsense.ini ] ---------
..

- [ File = ./tests/su.ini ] ---------
.....

- [ File = ./tests/audit_recon.ini ] ---------
..

- [ File = ./tests/aws_s3_access.ini ] ---------
.......
```

</p>
</details>

- [X] 4.d New CDB lists must include the proper test entry in the correct .ini file.

### E2E testing

- ~~5.a Logs for new or modified decoder/rules sent to a manager running with this PR ruleset appear in Kibana.~~ 
- ~~5.b There are not affected or broken visualizations on Kibana.~~
- ~~5.c New or modified items can be seen correctly using APP ruleset navigation.~~

### Elasticsearch Template

- [X] 6.a Known fields with output format managed and usually used for searching are included in template array index.query.default_field. 
- [X] 6.b The new field with the correct date format is stored as a "date" type field in the template.
- [X] 6.c The new extracted IP fields are in the pipeline as "geo" field and "geo_point" type in the template. 
- [X] 6.d Known fields with output format managed and usually used for searching are included in the template.

### Stoppers

- [x] 7.a No previous rule ID changes without triple check.
- [x] 7.b No previous decoder name changes without triple check.
- [x] 7.c No previous file name changes without triple check.
- [x] 7.d No previous test changes its 'rule' field value without triple check.

### Others

- [x] 8.a Each file has the correct copyright block.
- [x] 8.b The copyright block doesn't have "Author" only "Created by Wazuh". To include an "Author" request triple check.
- [x] 8.c The copyright block doesn't use "-". 
- [x] 8.d The rule files don't have any sample log.
- [x] 8.e The decoder file has sample logs next to the decoder that matches that log.
- [x] 8.f The decoder and rule files have information about software, version, and any helpful information.
- [x] 8.g The PR has a single commit with CHANGELOG changes in the correct format.
- [x] 8.h The new rule ID is not in use.
- [x] 8.i The new rule ID must be in the defined IDs range. 
- [x] 8.j New rules ID range must be verified with a triple check and noted in the rules ID document.
- [x] 8.k The new extracted IP fields are in the pipeline as "geo" field and "geo_point" type in the template.

### CODE REVIEW

File: ruleset/decoders/0300-sophos_decoders.xml [hunk 1]
```diff
@@ -1,87 +1,88 @@
...
-Sophos Anti-Virus
-SAV.txt
-https://community.sophos.com/kb/en-us/43391
-
-20160806 050000	Scan 'Sophos Cloud Scheduled Scan' started.
-20160806 052043	Scan 'Sophos Cloud Scheduled Scan' completed.
-20160805 175034	User (NT AUTHORITY\SYSTEM) has stopped on-access scanning for this machine.
-20160805 175143	Using detection data version 5.29 (detection engine 3.65.2). This version can detect 11628132 items.
- -->
+  Sophos decoders for:
```
Comment (72nomada) [inline ruleset/decoders/0300-sophos_decoders.xml:6]: remove vendor in **Decoders for:** line

File: ruleset/rules/0415-sophos_rules.xml [hunk 2]
```diff
@@ -1,56 +1,55 @@
...
-  -  Created by Wazuh, Inc.
-  -  Copyright (C) 2015-2021, Wazuh Inc.
-  -  This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+  Copyright (C) 2015-2021, Wazuh Inc.
 -->
 
-<!-- ID: 82100 - 82199 -->
-
+<!-- 
+  Sophos rules for:
```
Comment (72nomada) [inline ruleset/rules/0415-sophos_rules.xml:6]: Remove vendor from **Rules for:**

File: ruleset/rules/0415-sophos_rules.xml [hunk 3]
```diff
@@ -1,56 +1,55 @@
...
+  </rule>
+
+  <rule id="82105" level="3">
+    <if_sid>82100</if_sid>
+    <match>detection data version</match>
+    <description>Sophos database updated</description>
+    <group>gpg13_4.4,</group>
+  </rule>
+
+  <!-- Sophos Win rules-->
```
Comment (72nomada) [inline ruleset/rules/0415-sophos_rules.xml:50]: remove

**Comment by jcruzlp** [comment]
This PR will not implement changes in other files, just revert the changes added in the previous one. In order to achieve that will open a new PR for revert the changes, not rework them.

**Pull request closed without merging**